example/ndpiSimpleIntegration.c

```diff
@@ -1108,7 +1109,7 @@ int main(int argc, char ** argv)
   printf("usage: %s [PCAP-FILE-OR-INTERFACE]\n"
 	 "----------------------------------\n"
 	 "nDPI version: %s\n"
-	 " API version: %u\n"
+	 " API version: %" PRIu64 "\n"
```

Comment:
`PRIu64` may not work for all Windows CCs.
Better use `%llu` and cast (if necessary) the format param to `(unsigned long long int)`.

Reply:
How is it possible it may not work? It's the standard. No? Also, what is Windows CC ?

Reply:
[MSVC wants %I64u :/](https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-6.0/aa261215(v=vs.60)?redirectedfrom=MSDN)